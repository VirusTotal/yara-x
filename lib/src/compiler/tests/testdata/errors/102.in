rule test {
  strings:
    $a = { 11 [-1-65535] 22 }
  condition:
    $a
}