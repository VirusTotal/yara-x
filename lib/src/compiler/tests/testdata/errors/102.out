error: invalid integer
 --> line:3:16
  |
3 |     $a = { 11 [-1-65535] 22 }
  |                ^^ this number is out of the valid range: [0, 65535]
  |